apis/infix-en_GB/extensions/kotlin_1_3/atrium-api-infix-en_GB-kotlin_1_3-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/kotlin_1_3/keywords.kt

```diff
@@ -0,0 +1,14 @@
+@file:Suppress("ClassName")
+
+package ch.tutteli.atrium.api.infix.en_GB.kotlin_1_3
+
+import ch.tutteli.atrium.api.infix.en_GB.Keyword
+import ch.tutteli.atrium.api.infix.en_GB.toBe
+
+interface Keyword
+
+/**
+ * Represents the pseudo keyword `success` as in [toBe] `success`
+ * It can be used for a parameter less function so that it has one parameter and thus can be used as infix function
```

Comment:
```suggestion
 * It can be used for a parameter less function so that it has one parameter and thus can be used as infix function.
 *
 * @since 0.11.0
```